WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -346,6 +357,31 @@ private extension AddProductCoordinator {
         navigationController.pushViewController(viewController, animated: true)
     }
 
+    /// Presents AI Product Creation survey
+    ///
+    func presentProductCreationAIFeedbackIfApplicable() {
+        let useCase = ProductCreationAISurveyUseCase()
+        useCase.numberOfTimesAIProductCreated += 1
```

Comment:
Nit: can we add a helper method to increase this value instead of keeping the property accessible and mutable from outside?

Reply:
Good suggestion. Done in https://github.com/woocommerce/woocommerce-ios/pull/11390/commits/4602edb70ba264d9692ecfda2c6b65475fd3f4ca